addons/core/intercomConfig/config.cpp

```diff
@@ -15,19 +15,19 @@ class CfgPatches {
 
 #define IntercomMacro class ACE_SelfActions : ACE_SelfActions { \
     class TFAR_IntercomChannel { \
-        displayName = "IntercomChannel"; \
+        displayName = "$STR_TFAR_Intercom_ACESelfAction_Name"; \
         condition = "true"; \
         statement = ""; \
         icon = ""; \
         class TFAR_IntercomChannel_1 { \
-            displayName = "Cargo"; \
-            condition = "true"; \
-            statement = "(vehicle ACE_Player) setVariable [format ['TFAR_IntercomSlot_%1',(netID ACE_Player)],0,true];"; \
+            displayName = "$STR_TFAR_Intercom_ACESelfAction_Channel1"; \
+            condition = QUOTE(((vehicle ACE_Player) getVariable [ARR_2(Intercom_Variable,0)]) != 0); \
```

Comment:
wtf is this? The only stringtable related change that should be in here is the displayName.

Reply:
Oh, this might I try to compare with your latest commit some time before.
Like this. So shoud I restore this change back to old one ? Just change displayName ?
https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/cb38cbc597a04fbbcbc86eac9eb2a02c0cfd3a70/addons/core/intercomConfig/config.cpp#L26

Reply:
yes